<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="net04-PC" ignored="0" name="Test" tests="8" failures="2" timestamp="29 Jan 2018 06:26:20 GMT" time="0.13" errors="0">
  <testcase name="Test1" time="0.01" classname="Test.t">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.01" classname="Test.t">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="Test.t">
    <skipped/>
  </testcase> <!-- Test1 -->
  <testcase name="Test1" time="0.0" classname="Test.t">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Test.t.Test1(t.java:12)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.0" classname="Test.t">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="Test.t">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.0" classname="Test.t">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test2" time="0.01" classname="Test.t">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Test.t.Test2(t.java:18)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Test -->
